electrum.php

```diff
@@ -105,6 +109,88 @@ public function createnewaddress() : string {
 		return $response;
 	}
 
+	/**
+	 * Gives a JSON array of all current addresses.
+	 * 
+	 * @return array List of addresses
+	 */
+	public function listaddresses() : array {
```

Comment:
Wood be cool if `listaddresses()` function implemented *optional* funded boolean parameter.
No need in doing this, just saying my thoughts.